app/models/user.rb

```diff
@@ -66,7 +68,11 @@ def multiple_identities?
   end
 
   def active_profile
-    profiles.where(active: true).first
+    profiles.find(&:active?)
+  end
+
+  def verified?
+    active_profile ? true : false
   end
```

Comment:
I guess I'm wrong about the name "decorator": https://robots.thoughtbot.com/evaluating-alternative-decorator-implementations-in

http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/ is also good.

I still think, though, that we've not been consistent about using the decorator pattern to date, and there are lots of methods in the current User model that should be moved.